velox/functions/sparksql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -206,6 +206,22 @@ TEST_F(DateTimeFunctionsTest, weekOfYear) {
   EXPECT_EQ(8, weekOfYear("2008-02-20"));
   EXPECT_EQ(15, weekOfYear("2015-04-08"));
   EXPECT_EQ(15, weekOfYear("2013-04-08"));
+
```

Comment:
Locally tried this test while it fails. Would you take a look?

> [  FAILED  ] 1 test, listed below:
[  FAILED  ] DateTimeFunctionsTest.weekOfYear

Reply:
The local test of the new commit passed. Please help check again, thank you.